igor-web/src/main/groovy/com/netflix/spinnaker/igor/service/BuildServices.java

```diff
@@ -46,4 +47,8 @@ public BuildService getService(String name) {
             .sorted()
             .collect(Collectors.toList());
     }
+
+    public Map<String, BuildService> getAllBuildServices() {
```

Comment:
As noted above, I'd prefer if any logic/processing of build services could happen directly in this class, with the result returned to callers.  It looks like this could be replaced with a function that returns a `List<BuildServiceView>`.

Prior to a recent refactor I did, most callers were just grabbing the map and doing their own (often duplicated) logic so I'd like to avoid adding a public method here that just returns the map, in favor of having callers ask for the data they want.

Reply:
Fixed in 4bc8b65ea8083dfe7922eab43fd67dbd6f20f7fd

Reply:
Looks great, thanks!